src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Requests/FileUploadUpdateLogRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests
+{
+    public class FileUploadUpdateLogRequest : ApimSaveDataRequest
+    {
+        public long ProviderId { get; set; }
+        public long LogId { get; set; }
+    }
+}
```

Comment:
we could remove this (as it's not yet needed)

Reply:
yep removed as part of mass removal of update endpoint